tests.py

```diff
@@ -49,8 +49,11 @@ def test_get_sld_from_empty_list(self):
         psl = publicsuffix.PublicSuffixList([])
         assert 'com' == psl.get_sld('com')
         assert 'com' == psl.get_sld('COM')
-        assert 'com' == psl.get_sld('.com')
-        assert 'com' == psl.get_sld('a.example.com')
+        # '.com' -> <empty>.<com> -> None, empty labels are not allowed
+        assert None == psl.get_sld('.com')
```

Comment:
since this is the safe, not unsafe version, my expectation is that the dots are stripped. is this not the case? so '.com' becomes 'com' before being processed. it doesn't have a second label, but it's not empty either. previously we'd return 'com' for this. 

the sld from empty list is essentially returning two labels. there are use cases for this, even though it seems like a bizarre thing to do. so the example.com example is good here too.